eng/Version.Details.xml

```diff
@@ -35,16 +35,16 @@
       <Sha>0f7f38c4fd323b26da10cce95f857f77f0f09b48</Sha>
     </Dependency>
     <Dependency Name="System.IO.Pipelines" Version="4.7.2" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
-      <Uri>https://github.com/dotnet/corefx</Uri>
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-corefx</Uri>
       <Sha>059a4a19e602494bfbed473dbbb18f2dbfbd0878</Sha>
     </Dependency>
-    <Dependency Name="System.Net.Http.WinHttpHandler" Version="4.7.2" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="System.Net.Http.WinHttpHandler" Version="4.7.0" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
Something **bad** is happening with strict coherency. https://github.com/dotnet/core-setup/blob/f96e1b909b3668c773f4d2f8c16a0633c8335409/eng/Version.Details.xml#L125 shows version 4.7.2 but this PR doesn't seem to be using that. What's going on @wtgodbe @mmitche❔❕❔

Reply:
Strict coherency isn't actually enabled inside Maestro. It requires all the work currently in progress to enable. This is still old coherency. That said, if a repo is strict coherency compatible, it should generally have the same update as non-strict coherency.

Reply:
I'm debugging because I think it should be picking what is in core-setup (there shouldn't be anything newer)

Reply:
Alright, this is because the way that builds that contributed to a given dependency graph node are looked up based on the repo URL as well as the sha.  Which means that coherency doesn't even find the 4.7.2 version.

https://github.com/dotnet/core-setup/blob/f96e1b909b3668c773f4d2f8c16a0633c8335409/eng/Version.Details.xml#L126

This build was actually internal. So, the repo URL should be the corefx internal one, at which point this repo would update to the proper version.

Reply:
Let's watch to see if dotnet/core-setup#9069 triggers an update here that avoids the downgrade

Reply:
That is, don't merge without subscription updating to versions that include that fix.

Reply:
Downgrade resolved 😺